app/code/Magento/Security/Model/Plugin/AccountManagement.php

```diff
@@ -63,10 +73,21 @@ public function beforeInitiatePasswordReset(
         $template,
         $websiteId = null
     ) {
-        $this->securityManager->performSecurityCheck(
-            $this->passwordRequestEvent,
-            $email
-        );
+        $originalScope = $this->scope->getCurrentScope();
+        $this->scope->setCurrentScope(\Magento\Framework\App\Area::AREA_FRONTEND);
```

Comment:
Does this work with multiple stores each with a different value?

Reply:
This has many considerations, I'll explain them in a separate comment in the main thread.